docs-v2/spec.yaml

```diff
@@ -803,6 +803,10 @@ paths:
                                                             type: string
                                                             description: The timestamp at which Nango last detected a change to this record
                                                             example: "2023-09-18T15:20:35.941305+00:00"
+                                                        cursor:
+                                                            type: string
+                                                            description: The current cursor of the record. Use this to fetch records updated after this record
```

Comment:
is this really `>` or `>=`? Looking at the code, I feel like it's not after this record but "at this record" which is a detail but change things when you consume

Reply:
It is after this record:

```
const decodedCursorValue = Buffer.from(cursorValue, 'base64').toString('ascii');
const [cursorSort, cursorId] = decodedCursorValue.split('||');

query = query.where((builder) =>
                builder.where('updated_at', '>', cursorSort).orWhere((builder) => builder.where('updated_at', '=', cursorSort).andWhere('id', '>', cursorId))
            );
```

Greater than the created_at

Reply:
ah okay I didn't saw the query, thanks 👌🏻